go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	github.com/prometheus/common v0.9.1
 	github.com/prometheus/tsdb v0.8.0 // indirect
 	github.com/rancher/dynamiclistener v0.2.1-0.20200418023342-52ede5ec9234
-	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200512184531-c395feca5b3b
+	github.com/rancher/kontainer-engine v0.0.4-dev.0.20200327180925-6d88ca2af46f
 	github.com/rancher/machine v0.15.0-rancher25
-	github.com/rancher/norman v0.0.0-20200417060332-5c2a13443bb8
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
 	github.com/rancher/rdns-server v0.0.0-20180802070304-bf662911db6a
 	github.com/rancher/remotedialer v0.2.6-0.20200401224317-04cc22157b83
-	github.com/rancher/rke v1.1.1
+	github.com/rancher/rke v1.1.0-rc9.0.20200327175519-ecc629f2c3d5
 	github.com/rancher/security-scan v0.1.7-0.20200222041501-f7377f127168
-	github.com/rancher/steve v0.0.0-20200416160046-0a5d0fbca1bc
-	github.com/rancher/system-upgrade-controller v0.3.1
-	github.com/rancher/types v0.0.0-20200410170911-a4f6ba0aa09d
-	github.com/rancher/wrangler v0.5.4-0.20200417162130-9817cb5fdf6b
-	github.com/rancher/wrangler-api v0.5.0
+	github.com/rancher/steve v0.0.0-20200326205851-420f62f642eb
+	github.com/rancher/system-upgrade-controller v0.4.1-0.20200326220202-4655d4a551bd
+	github.com/rancher/types v0.0.0-20200327203838-a36e8d24795f
+	github.com/rancher/wrangler v0.5.4-0.20200326191509-4054411d9736
```

Comment:
Need to update this after merging https://github.com/rancher/wrangler/pull/93 with the commit from the release/v2.4 branch